Vagrantfile

```diff
@@ -77,6 +77,17 @@ def mount_folders(config, mount_options)
 end
 
 Vagrant.configure(2) do |config|
+  config.vm.boot_timeout=600
+  config.vm.provider :virtualbox do |v|
+    v.customize ["modifyvm", :id, "--uart1", "0x3F8", "4"]
+    v.customize ["modifyvm", :id, "--uartmode1", "file", File::NULL]
+    v.customize ["modifyvm", :id, "--audio", "none"]
```

Comment:
From my tests, these modifications are not really necessary. The first two lines with uart1 and uartmode1 seem to fix the freeze on their own.